### DISCUSSION TITLE
Unable to download extension on VSCode

### DISCUSSION THREAD

**Discussion by Webmekanic** [microsoft/vscode-discussions#842] [Extension Development QnA] [answered]
<img width="501" alt="Screenshot 2023-09-25 at 5 37 43 PM" src="https://github.com/microsoft/vscode-discussions/assets/81490414/7c2bc329-59f6-4f30-8332-33e7e91e1892">

**Comment by alefragnani** [comment, marked as answer]
Hi @Webmekanic ,

This can be caused by a few things like Proxies, network instability, user permission and so on, so I suggest you to look for the **log** in that message, trying to identify error. If you have no success, look for similar issues in the VS Code repo https://github.com/microsoft/vscode/issues (which has a good probability), but if you can't find it, open a new one (https://github.com/microsoft/vscode/issues/new?assignees=&labels=&projects=&template=bug_report.md) and add the provided log to help the VS Code team to identify the error.

By the way, for future issues about VS Code usage, please use the VS Code repository instead (https://github.com/microsoft/vscode/issues/new/choose). This Discussions repo is focused for extension development for now.

Hope this helps

**Comment by olguzzar** [comment]
Closing this as off-topic since this forum is specifically for extension development.
